sal_interfaces/MTAlignment/MTAlignment_Commands.xml

```diff
@@ -81,4 +145,92 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_setReferenceGroup</EFDB_Topic>
+    <item>
+      <EFDB_Name>referenceGroup</EFDB_Name>
+      <Description>Nominal point group to locate tracker station to and provide data relative to</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_halt</EFDB_Topic>
+    <item>
+      <EFDB_Name>halt</EFDB_Name>
+      <Description>halts any executing measurement plan and returns to ready state</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_setWorkingFrame</EFDB_Topic>
+    <item>
+      <EFDB_Name>workingFrame</EFDB_Name>
+      <Description>attempt to set the passed string as the SpatialAnalyzer working frame</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_loadSATemplateFile</EFDB_Topic>
+    <item>
+      <EFDB_Name>file</EFDB_Name>
+      <Description>SA Template file path and name. This is in the filesystem on the T2SA host.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_measureDrift</EFDB_Topic>
+    <item>
+      <EFDB_Name>pointgroup</EFDB_Name>
+      <Description>nominal pt group name in the default A collection to measure drift against.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_resetT2SA</EFDB_Topic>
+    <item>
+      <EFDB_Name>resetT2SA</EFDB_Name>
+      <Description>reboots T2SA and SA components</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_newStation</EFDB_Topic>
+    <item>
+      <EFDB_Name>newStation</EFDB_Name>
+      <Description>A new station is added and made to be the active instrument.</Description>
```

Comment:
Please use an active voice, e.g. "new station to be added and to be the active instrument". Please also drop the trailing period (or add one to all other descriptions in this XML file).